src/scripts/generate_covid_case_datamart.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import pathlib
+import json
+from typing import List, Callable
+from utils import read_blob, record_combination_func, write_blob
+
+
+def standardize_lab_result(result: str) -> str:
+    """
+    Given a covid test result and dictionary whose keys (POSITIVE, NEGATIVE, and
+    INCONCLUSIVE) represent standard results and whose values are lists containing
+    non-standard results that map to the associated standard result then return a
+    standardized result. Return null is no standard result could not be found.
+    """
+    standard = json.loads(
+        open(
+            pathlib.Path(__file__).parent / "assets" / "covid_lab_results_standard.json"
+        ).read()
+    )
+    result = result.upper()
+    for standard_result, non_standard_results in standard.items():
+        if result in non_standard_results:
+            return standard_result
```

Comment:
We could also make this code faster and more efficient by inverting the dictionary before we process the result. If we have a dict of the form `non-standard-result --> standardized result` then we don't have to loop over the iterators inside the dictionary for every result we process. Inverting a dictionary is also super easy, so in conjunction with my comment above about moving the dict loading to outside of whatever loop we do, let's just add some code to invert, e.g.

```
nonstandard_to_standard_mappings = {}
for std_res in standard.keys():
    for nonstd_res in standard[std_res]:
        nonstandard_to_standard_mappings[nonstd_res] = std_res
```